build.assets/Dockerfile-centos7-fips

```diff
@@ -46,12 +46,17 @@ RUN yum groupinstall -y 'Development Tools' && \
     yum install -y \
     #required by libbpf
     centos-release-scl \
+    cmake3 \
+    # required by BoringSSL
+    devtoolset-7-* \
     # required by libbpf
     devtoolset-11-* \
     # required by libbpf
     elfutils-libelf-devel-static \
     git \
     net-tools \
+    ninja-build \
```

Comment:
You're right. I couldn't find a specific version of ninja that is required, but now I've found the document that precisely describes the tools and their versions. It's in the section `12. Guidance and Secure Operation` of https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp3678.pdf

They also mention that go version has to be 1.12.7 for the compilation of the BoringSSL. That made me think of moving compilation of the BoringSSL library into separate stage (like libbpf) and then just copy compiled libraries into the last stage. That would make clear spearation of tools and it would be clear why we install specific versions of them. What do you think about it? @jakule